target/linux/mvebu/modules.mk

```diff
@@ -2,11 +2,7 @@ define KernelPackage/linkstation-poweroff
   SUBMENU:=$(OTHER_MENU)
   DEPENDS:=@TARGET_mvebu
   TITLE:=Buffalo LinkStation power off driver
-  KCONFIG:= \
-	CONFIG_POWER_RESET=y \
-	CONFIG_POWER_RESET_LINKSTATION \
-	CONFIG_POWER_RESET_QNAP=n
-
+  KCONFIG:=CONFIG_POWER_RESET_LINKSTATION
```

Comment:
On a quick look, I wonder whether this and the kernel config changes should be in a separate commit ...

Reply:
Yeah, that would be better.